### ISSUE TITLE
Whether netbird has functions similar to tailscale Mullvad VPN egress node？

### ISSUE THREAD

**Issue by Lotusrootsoup** [netbirdio/netbird#1574]
Whether netbird has functions similar to tailscale Mullvad VPN egress node？

**Comment by Lotusrootsoup** [comment]
Thank you for your answer and look forward to your new features

**Comment by Lotusrootsoup** [comment]
> 你好@Lotusrootsoup，我们还没有，但这是经常要求的功能，请参阅 [#1138 ](https://github.com/netbirdio/netbird/issues/1138) [#289 ](https://github.com/netbirdio/netbird/issues/289) #528
> 
> 我想确认这是我们公共路线图和 2024 年 3 月计划工作的一部分 https://github.com/netbirdio/netbird/projects/2

Thank you for your answer and look forward to your new features!

**Comment by nazarewk** [comment]
@Lotusrootsoup I believe it can be closed due to introduction of dedicated Exit Nodes?

**Comment by shuaibmunshi** [comment]
@nazarewk dedicated exit nodes are not a replacement for the feature being requested here. Tailscale's mullvad integration turns tailscale into a coordination service and your internet-bound traffic exits through mullvad's exit nodes automatically. In order to do the same with netbird presently you need to turn one of your own nodes into a wireguard peer node and do the routing between the VPNs yourself. 

https://mullvad.net/en/blog/2023/9/7/tailscale-has-partnered-with-mullvad

**Comment by tabalugadreieck** [comment]
I want to add that I would also really like to see this feature. Be it mullvad or some other privacy VPN provider. Its the only thing that keeps me on tailscale atm.
Having netbird handle the authentication with the privacy VPN provider adds another stage that makes it even more difficult to identify you than just using a privacy VPN standalone.
Is there some roadmap for it?

**Comment by oldgit27** [comment]
> I want to add that I would also really like to see this feature. Be it mullvad or some other privacy VPN provider. Its the only thing that keeps me on tailscale atm. Having netbird handle the authentication with the privacy VPN provider adds another stage that makes it even more difficult to identify you than just using a privacy VPN standalone. Is there some roadmap for it?

Likewise this is the only missing feature that prevents me from switching from Tailscale to Netbird.  The rest looks superb.

**Comment by chukfinley** [comment]
+1!

**Comment by BarkBarklington** [comment]
has this issue been resolved or worked on?

**Comment by BLACK4585** [comment]
I also want to jump in here and show my support for this.
With all the political headaches in many countries going on, Germany planning a law to force ISPs and service providers to save IP addresses for 3 moths and discussions across different countries about the anonymity in the internet, I think this evolves day by day to a must-have.
Anonymity is often crucial for normal people to just stay anonymous, journalists to uncover their stories, NGOs to keep doing their work and many many cases more.
TLDR: VPNs to hide the geolocation and personal data are becoming more important every day.

And especially we tech-savyy people running our own infrastructure across different geolocations should have the ability to surf the web as power users and still stay connected to our self hosted services.

_thank you for listening to my ted talk_

**Comment by JacobWennebro** [comment]
I would like to see this added as well

**Comment by castor-homelab** [comment]
Would love to see the functionality for Netbird to work with MullvadVPN as an exit node (with kill switch if traffic cannot be routed through it).

@surik - I can no longer see this on the roadmap but as far as I know it has not been implemented. Will it be considered again at some point?

**Comment by dyld-w** [comment]
Commenting in support as I'd really like to see this as well: https://tailscale.com/docs/features/exit-nodes/mullvad-exit-nodes. Thanks!

**Comment by cchance27** [comment]
Is this going to be something coming to netbird, having generic wireguard exitnodes like mullvad would be so helpful
